lib/translator/index.js

```diff
@@ -29,13 +29,18 @@ function Translator(acceptLanguage, allTranslations, logger = console) {
 
     const locales = LocaleParser.getLocales(acceptLanguage);
     const languages = Transformer.transform(allTranslations, locales, DEFAULT_LOCALE, this.logger);
+
     /**
      * @private
      * @type {string}
      */
     this._locale = LocaleParser.getPreferredLocale(locales, languages, DEFAULT_LOCALE);
 
-    this.setLanguage(languages);
+    /**
+     * @private
+     * @type {Object.<string, string>}
```

Comment:
I think you don't need those statements here anymore

Reply:
I'm just reverting back to what it was - if we want to remove these, let's do it consistently for all of them in a separate change.